scripts/stairwell_install_win.ps1

```diff
@@ -0,0 +1,36 @@
+<#
+    Use this script to install the Stairwell forwarder on Windows OS machines.
+    This script requires your file forwarder token and environment id.
+    Visit https://docs.stairwell.com for further assistance.
+
+    Typically this would be used for remotely installing, such as in an Azure scale set
+    ex. Set-AzVMCustomScriptExtension -ResourceGroupName "ResourceGroup11" -Location "Central US" -VMName "VirtualMachine07" -Name "StairwellScriptExt" -FileName "stairwell_install_win.ps1" -Argument 'ForwarderToken EnvironmentId' | Update-AzureVM
+#>
+param(
+    [Parameter(Mandatory=$True)]
+    [string]$ForwarderToken,
+
+    [Parameter(Mandatory=$True)]
+    [string]$EnvironmentId
+)
+
+# Check for C:\Temp directory
+if (!(Test-Path "C:\Temp")) {
+    New-Item -ItemType Directory -Path "C:\Temp"
+}
+
+# Try to download forwarder
+try {  
+    $ProgressPreference = 'SilentlyContinue'  
+    Invoke-WebRequest -Uri "https://downloads.stairwell.com/windows/latest/InceptionForwarderBundle.exe" -UseBasicParsing -OutFile "C:\Temp\InceptionForwarderBundle.exe"
```

Comment:
this is pointing to the latest in the 1.3.X stream, but i we can start uploading the latest 1.4.X, `StairwellForwarderBundle.exe` to the same bucket